package.json

```diff
@@ -4,6 +4,8 @@
   "version": "1.0.1",
   "author": "YLD <hello@yld.io>",
   "dependencies": {
+    "@mdx-js/mdx": "^1.5.1",
```

Comment:
can you check if we need all these?

Reply:
Removed gatsby-remark-prismjs, everything else we need